airbyte-integrations/connectors/destination-dev-null/src/test-integration/kotlin/io/airbyte/integrations/destination/dev_null/DevNullBasicFunctionalityIntegrationTest.kt

```diff
@@ -26,4 +26,9 @@ class E2eBasicFunctionalityIntegrationTest :
     override fun testBasicWrite() {
         super.testBasicWrite()
     }
+
+    @Test
```

Comment:
are the overrides necessary for the tests to run?

Reply:
It's not. I assumed I'd gotten out of sync with you at some point and was missing a dependency. I can remove it